packages/angular/src/errorhandler.ts

```diff
@@ -32,15 +32,17 @@ class SentryErrorHandler implements AngularErrorHandler {
       ...options,
     };
   }
-
   /**
    * Method called for every value captured through the ErrorHandler
    */
   public handleError(error: unknown): void {
     const extractedError = this._extractError(error) || 'Handled unknown error';
 
     // Capture handled exception and send it to Sentry.
-    const eventId = runOutsideAngular(() => Sentry.captureException(extractedError));
+    const angularVersion = VERSION && VERSION.major ? parseInt(VERSION.major, 10) : 0;
+    const eventId = runOutsideAngular(() =>
+      Sentry.captureException(extractedError, { contexts: { angular: { version: angularVersion } } }),
+    );
```

Comment:
Great idea, thanks! will change it!

Reply:
About entire vs major: Not sure what's better for data querying. I think for the particular use case why we're doing this, the major should be enough but overall it might be worth collecting the full version (which begs the question if we should collect it in one string or in major, minor, patch properties). Do you know what's the better way of handling this (w.r.t indexing/querying in looker)?

Reply:
Realized this didn't comment 😓 

I think let's stick with the major for now since we get the most insights out of it. We can change this in the future if needed.